src/main/java/JavaQuestionTwo/AddressCompare.java

```diff
@@ -0,0 +1,23 @@
+package JavaQuestionTwo;
+
+import java.util.Comparator;
+
+public class AddressCompare implements Comparator<user>{
+	int asc;
+	int result;
+	
+	public AddressCompare(int asc) {
+		this.asc=asc;
+	}
+	
+	@Override
+	public int compare(user u1, user u2) {
```

Comment:
not required.
use [lamda expression](https://www.baeldung.com/java-8-sort-lambda)

Reply:
Done